lib/plugins/create/templates/aws-nodejs-typescript/package.json

```diff
@@ -7,6 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
+    "@types/node": "^8.0.57",
     "serverless-webpack": "^3.0.0",
```

Comment:
@jamestharpe Thanks for the improvement. This is ok from my side. Although it is not in the scope of the changes, could you please also update `serverless-webpack` to `^4.0.0` to have it current?

Reply:
Done. While testing, I got an error related to including `"@types/aws-lambda": "0.0.22"` in my test project:

```
ERROR in ~/my-project/node_modules/aws-sdk/lib/config.d.ts
(39,37): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
```

I rolled back `serverless-webpack`, tried again, and still got the same error, so I'm not sure it was related. The fix was to include the following in my tsconfig.json `compilerOptions` value:

```json
"lib": [
      "es5",
      "es2015.promise"
    ]
```

I left it out of this PR, since I don't think it's related, but wanted to document it here just in case. I'd be happy to include it in the PR if you think it wise.

Reply:
It seems that this is a common problem with the aws types:
https://stackoverflow.com/questions/43119163/typescript-error-ts2693-promise-only-refers-to-a-type-but-is-being-used-as